src/compiler/output-targets/dist-custom-elements/index.ts

```diff
@@ -96,15 +96,15 @@ export const getBundleOptions = (
 /**
  * Get bundle options for rollup, run the rollup build, optionally minify the
  * output, and write files to disk.
- * @param config user-supplied Stencil configuration
+ *
+ * @param the validated Stencil configuration we're using
```

Comment:
Can we fix this inline???
```suggestion
 * @param config the validated Stencil configuration we're using
```